docs/get-started/sync-client-quick-start.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Planet Client Quick Start
+---
+
+The Planet SDK for Python makes it easy to access Planet’s massive repository of satellite imagery and add Planet
+data to your data ops workflow.
+
+**Note:** This is the new, non-asyncio client. If you want to take advantage of asyncio, check the [asyncio client quick start guide](quick-start-guide.md).
+
+Your feedback on this version of our client is appreciated. Please raise an issue on [GitHub](https://github.com/planetlabs/planet-client-python/issues) if you encounter any problems.
+
+## Dependencies
+
+This package requires [Python 3.8 or greater](https://python.org/downloads/). A virtual environment is strongly recommended.
```

Comment:
nit:  The SDK is pinned to Python v >=3.9 now https://github.com/planetlabs/planet-client-python/blob/main/pyproject.toml#L19

Reply:
Resolving, I pushed a commit to update this to 3.9.